crimsobot/cogs/mystery.py

```diff
@@ -28,6 +28,27 @@ async def tarot(self, ctx: commands.Context) -> None:
         # if no subcommand provided, give a three-card reading
         await self.ppf(ctx)
 
+    @tarot.command(name='one', brief='Get a single card.')  # something better needs to be put here, i do not know much
```

Comment:
`'A single reading'`, perhaps?

Reply:
sounds good, changed in https://github.com/crimsobot/crimsoBOT/pull/73/commits/12571c60232c4e2280f173bf90870845d1ac6791